docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,485 @@
+
+```bash
+ayx-sdk-cli plugin run DanceableLyrics -o danceable.csv
+```
+
+The above command will run our plugin and produce `danceable.csv`. You may run `ayx-sdk-cli plugin run --help` for additional options.
+
+Given the command above, our output should look something like this (I am using [Rich CLI](https://github.com/Textualize/rich-cli) to preview):
+
+![danceable.csv](./assets/danceable-csv.png)
+
+# Install and Run in Designer
+In this section we will go over the two ways to install the plugin into Designer.
+
+## Method 1
+After you create a .yxi, you can double-click the .yxi to install it into Designer. This opens Designer and prompts you to install the package in a new dialog box. It looks something like this:
+
+![Designer Install Prompt](./assets/designer-install-prompt.png)
+
+Once it installs, you can find the plugin under the `Python SDK Examples` tool tab.[^2]
+
+## Method 2
+You can also create the `.yxi` _**and**_ install it all at once via the `ayx_plugin_cli designer-install` command. Choose the install option that matches your Designer install. Typically, this is the `user` install option.
+
+```bash
+~/MyWorkspace$ ayx_plugin_cli designer-install
+Install Type (user, admin) [user]: user
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+[Creating YXI] -- generate_config_files:generate_tool_config_xml
+[Creating YXI] .  generate_config_files:generate_manifest_jsons
+[Creating YXI] Generating manifest.json for DanceableLyrics...
+[Creating YXI] Done!
+...omitted...
+[Creating YXI] finished
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] started
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] .  install_yxi
+[Installing yxi ~\MyWorkspace\build\yxi\DanceableLyrics.yxi into designer] finished
+If this is your first time installing these tools, or you have made modifications to your ayx_workspace.json file, please restart Designer for these changes to take effect. # <-- Done installing into Designer!
+```
+
+Once the command finishes you can open Designer and find your tool under the `Python SDK Examples` tab.[^1]
+
+# Run in Designer
+When you run this tool in Designer it yields the following output:
+
+![Designer Output](./assets/designer-output.png)
+
+You can see that our top danceable track with the input lyrics provided is ["Track 10" by The Procussions](https://open.spotify.com/track/1xZJKDCFXuCRUr82qzaRDp)!
+
+https://open.spotify.com/track/1xZJKDCFXuCRUr82qzaRDp
+
+Thanks for reading, I hope you enjoyed this tutorial!
+
+
+> [^1]: :warning: If you created the plugin workspace with a non-default `Tool Category` (from the [Create a Workspace](#1-create-a-workspace) section), then the plugin will show up in the tab that corresponds to the input that was passed to `Tool Category`.
```

Comment:
TW review:

> [^1]: :warning: If you created the plugin workspace with a non-default `Tool Category` (from the [Create a Workspace](#1-create-a-workspace) section), then your plugin will appear in the tool category that corresponds to the input that you passed to `Tool Category`.

Reply:
Not sure on this one?

Reply:
thanks for the heads up. It wasn't rendering my comment for some reason. I got it to show up now - let me know if you have any questions.